### ISSUE TITLE
Drag & Drop to any app

### ISSUE THREAD

**Issue by 7OH** [M2Team/NanaZip#230]
Now this app (like 7zip too) support drag&drop from arch only to explorer.
Please - make it possible to drop files to any app - especially to TotalCmd

**Comment by LongDono** [comment]
Yes, it is very frustrating to drag and drop a compressed file on to a destination directory and only see Copy here/Move here/Create shortcuts here when we are used to seeing Unzip here...
